foundry.toml

```diff
@@ -0,0 +1,12 @@
+[profile.default]
+src = 'src'
+out = 'out'
+libs = ['lib']
+fs_permissions = [{ access = "read-write", path = "./"}]
```

Comment:
why do we want/need to turn on foundry read-write access to the project?

Reply:
This is necessary for the code injection in the `ArbitrumDomain` contract. Try removing it, run the tests and you will see the error.

Reply:
(it doesn't seem like i can "resolve" comments here for some reason, but thanks, this makes sense)

Reply:
@iamchrissmith just sent you invite to the repo, you were missing writing access.